src/style.css

```diff
@@ -1 +1,84 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+.search-bar {
+  font-family: Avenir, sans-serif;
+  display: block;
+  box-sizing: border-box;
+  width: 100%;
+
+  margin-top: 20px;
```

Comment:
According to the checklist, be consistent with your margins. Choose either top or bottom margins for elements, but not both, to avoid potential margin collapse issues.